booking-engine-api/operations/services.md

```diff
@@ -0,0 +1,268 @@
+            "TotalAmount": { /* Amount */ },
+            "AverageAmountPerTimeUnit": { /* Amount */ }
+          }
+        },
+        {
+          "MinRateId": "f02923a9-ffa3-4d83-a9d7-abc800c82222",
+          "MinPrice": {
+            "TotalAmount": { /* Amount */ },
+            "AverageAmountPerTimeUnit": { /* Amount */ }
+          },
+          "MaxPrice": {
+            "TotalAmount": { /* Amount */ },
+            "AverageAmountPerTimeUnit": { /* Amount */ }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `RateGroups` | array of [Rate group](hotels.md#rate-group) | required | Information about all available rate groups. |
+| `Rates` | array of [Rate](hotels.md#rate) | required | Information about all available rates. |
+| `CategoryPrices` | array of [Occupancy](#category-price) | required | Prices for all specified categories. |
+
+#### Category price 
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CategoryId` | string | required | Unique identifier of category. |
+| `OccupancyPrices` | array of [Occupancy](#occupancy) | required | List of Occupancies containing of age categories combination for which are prices applied for. |
+| `RateGroupPrices` | array of [Rate group price](#rate-group-price) | required | Prices for all specified categories and occupancies from Occupancy Prices. |
+
+#### Occupancy
+Objects composed of identifier `AgeCategoryId` referring to Age Category entity with `PersonCount` number of persons of this Age category type.
+
+#### Rate group price
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `MinRateId` | string | required | Unique identifier of rate. |
+| `MinPrice` | [Amount](#amount) objects | required | TotalAmount contains total price for the specified interval for rate, AverageAmountPerTimeUnit contains price for one timeUnit. (Day/Night) |
+| `MaxPrice` | [Amount](#amount) objects | required | TotalAmount contains total price for the specified interval for rate, AverageAmountPerTimeUnit contains price for one timeUnit. (Day/Night) |
+
+#### Rate
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the rate. |
+| `Name` | [Localized text](hotels.md#localized-text) | required | Name of the rate localized into all supported languages. |
+| `Description` | [Localized text](hotels.md#localized-text) | required | Description of the rate localized into all supported languages. |
+| `IsPrivate` | boolean | required | Set to `true` for promotion rate enabled by provided `VoucherCode`. |
+| `ServiceId` | string | required | Unique identifier of the [Service](configuration.md#service) to which the rate is bound. |
+| `RateGroupId` | string | required | [Rate group](hotels.md#rate-group) Identifier of the rate. |
+| `Ordering` | number | required | Number defining the ordering of the rate. |
+
+
+### Amount
```

Comment:
Should be #### Amount

This looks like an example of the content of the Amount object, but where is the object definition?